src/api/goods.js

```diff
@@ -1,11 +1,17 @@
 // eslint-disable-next-line
 const API_URL = `https://mate-academy.github.io/react_dynamic-list-of-goods/goods.json`;
+const getFetchGoods = () => fetch(API_URL).then(response => response.json());
```

Comment:
```suggestion
const fetchGoods = () => fetch(API_URL).then(response => response.json());
```
use only one verb
https://github.com/kettanaito/naming-cheatsheet